modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -405,14 +402,12 @@ definition {
 				pluralLabelName = "Custom Objects 161922");
 
 			ObjectAdmin.addObjectFieldViaAPI(
-				fieldBusinessType = "Attachment",
+				fieldBusinessType = "Text",
 				fieldLabelName = "Custom Field",
 				fieldName = "customObjectField",
-				fieldType = "Long",
-				fileSource = "userComputer",
+				fieldType = "String",
 				isRequired = "false",
-				objectName = "CustomObject161922",
-				showFilesInDocumentsAndMedia = "true");
+				objectName = "CustomObject161922");	
```

Comment:
Why are you changing the existing field type here?

Reply:
Because the macro **addObjectSingleFieldEntryViaJSONAPI** that I'm using later on don't support attachment fields.